addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,102 @@
+oc create -f xpaas-templates/eap70-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/eap70-sso-s2i.json -n openshift
+
+oc create -f xpaas-templates/jws30-tomcat7-basic-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-https-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat7-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-basic-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-https-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mongodb-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/jws30-tomcat8-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/karaf2-camel-amq-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-log-template.json -n openshift
+oc create -f xpaas-templates/karaf2-camel-rest-sql-template.json -n openshift
+oc create -f xpaas-templates/karaf2-cxf-rest-template.json -n openshift
+
+oc create -f xpaas-templates/openjdk18-web-basic-s2i.json -n openshift
+
+oc create -f xpaas-templates/processserver63-amq-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-amq-postgresql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-basic-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-mysql-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-persistent-s2i.json -n openshift
+oc create -f xpaas-templates/processserver63-postgresql-s2i.json -n openshift
+
+oc create -f xpaas-templates/spring-boot-camel-amq-template.json -n openshift
```

Comment:
These seem to be part of the fuse add-on as well. In fact the only thing missing from the fuse add-on are some karaf2 templates. What shall we do?

* Delete the fuse add-on?
  *  Ignoring Karaf2
  *  Adding Karaf2 to pass
* Keep both add-ons
   * As is
   * Rethink our use of 'oc create' and use 'oc apply'